tests/python_tests/helpers/param_config.py

```diff
@@ -87,20 +90,23 @@ def format_combination_sweep(
         for pack_dst in formats
     ]
 
+class TestParamsConfig(TypedDict):
+    testnames: str
```

Comment:
is this names or name?
```suggestion
    test_name: str
```